bin/lux

```diff
@@ -32,6 +33,14 @@ function inLuxProject() {
   }
 }
 
+function commandNotFound(cmd) {
+  console.log();
+  console.log(`  ${red(cmd)} isn't a valid command`);
+  console.log();
+  console.log(`  Type ${green('lux --help')} for a full list of commands`);
```

Comment:
Once again throwing stones in a glass house lol but what are your thoughts on changing the copy to the following?

> Use lux --help for a full list of commands